process/coordinator/process.go

```diff
@@ -33,7 +34,8 @@ type transactionCoordinator struct {
 	mutRequestedTxs sync.RWMutex
 	requestedTxs    map[block.Type]int
 
-	onRequestMiniBlock func(shardId uint32, mbHash []byte)
+	onRequestMiniBlock    func(shardId uint32, mbHash []byte)
```

Comment:
We should also inject RequestHandler here and drop the func pointer

Reply:
Agree. I think that the refactor of this Request mechanism should be done in one separate task.

Reply:
I have created a task for this https://elrondnetwork.atlassian.net/browse/EN-5226

Reply:
ok